src/serializers/shared.rs

```diff
@@ -217,7 +232,7 @@ impl BuildSerializer for CombinedSerializer {
         config: Option<&Bound<'_, PyDict>>,
         definitions: &mut DefinitionsBuilder<CombinedSerializer>,
     ) -> PyResult<CombinedSerializer> {
-        Self::_build(schema, config, definitions)
+        Self::_build(schema, config, definitions, true)
```

Comment:
It looks like `BuildSerializer` implementation for `CombinedSerializer` is completely pointless, you could just remove this trait implementation and just rename `_build` to `build`. That would remove the need for `build_base` and simplify code.

Reply:
The thing is I wanted to avoid changing the signature of `build`, so that I don't have to update all implementations to pass `use_prebuilt=true`.